sdmetrics/efficacy/base.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+from sklearn.model_selection import KFold
+
+from sdmetrics import Goal, Metric
+
+
+class MLEfficacy():
+
+    name = ""
+    target_table = ""
+    target_column = ""
+
+    metric_unit = ""
+    metric_goal = Goal.IGNORE
+    metric_domain = (float("-inf"), float("inf"))
+
+    def fit(self, X, y):
+        """This function implements a fit procedure which trains a binary
```

Comment:
This doc string seems to be from adversarial metrics. Needs an update to MLefficacy